pkg/es/writer/esBulkHandler.go

```diff
@@ -330,21 +386,37 @@ func ProcessIndexRequest(rawJson []byte, tsNow uint64, indexNameIn string,
 	bytesReceived uint64, flush bool, localIndexMap map[string]string, myid uint64,
 	rid uint64, idxToStreamIdCache map[string]string,
 	cnameCacheByteHashToStr map[uint64]string, jsParsingStackbuf []byte) error {
+	return nil
+}
+
+func ProcessIndexRequestPle(tsNow uint64, indexNameIn string,
+	bytesReceived uint64, flush bool, localIndexMap map[string]string, myid uint64,
+	rid uint64, idxToStreamIdCache map[string]string,
+	cnameCacheByteHashToStr map[uint64]string, jsParsingStackbuf []byte, pleArray []*writer.ParsedLogEvent) error {
+
+	for _, ple := range pleArray {
```

Comment:
technically the `indexNameIn` and the ple.GetIndexName()` should be exactly same , because when we did the batching, it was grouped based on the indexname.
Or is this here just for a safety check ?

Reply:
It was just an extra safety check in case we make more calls to this function later. Do you think I should remove it?